src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
-// write your code here
+const populationsEl = document.querySelectorAll('.population');
+const populations = [...populationsEl];
+
+const totalNum = populations
+  .map((el) => el.innerHTML.replaceAll(',', ''))
```

Comment:
The `replaceAll` method is used here to remove commas from the innerHTML of each element. Note that `replaceAll` is not supported in Internet Explorer. If you need to support older browsers, consider using `replace(/,/g, '')` instead.